### DISCUSSION TITLE
School-issued email address labeled as not

### DISCUSSION THREAD

**Discussion by parker02311** [community/community#112374] [GitHub Education]
### Hi everyone,

Hi, I have been using my school email address to attempt to reverify my student account but the email address is not working with the message `We require applicants to use a school-issued email address to apply.` despite me using a school-issued email address.

**Comment by samus-aran** [comment]
Hi @parker02311 👋 

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

You should provide evidence to support if your school has additional academic domains, has changed its academic email issuing policy, or does not provide academic emails. Reach out to your institution's IT team for this. 

We see that you have already reached out to the Education Support team. Please re-apply using your academic emails. Let us know how you get on. 👍

**Reply by parker02311** [reply]
@samus-aran hi, I have already gotten the domain fixed via a support ticket. The issue I am facing now is when I hit `Continue` instead of being redirected to the next step, the page just refreshes and gives the error `We require applicants to use a school-issued email address to apply.`
